src/main/resources/db/migration/V1_1_0__my_first_migration.sql

```diff
@@ -0,0 +1,66 @@
+
+create table role (
+    id serial primary key not null,
+    name varchar not null unique ,
+    is_admin boolean default false
+);
+
+create table users(
+    id serial primary key not null,
```

Comment:
I know this is not related to Java but as a programmer, you should add some indexes. They are important :smile: